samza-core/src/main/java/org/apache/samza/container/grouper/task/TaskNameGrouper.java

```diff
@@ -50,4 +51,8 @@
    * @return Set of containers, which contain the tasks that were passed in.
    */
   Set<ContainerModel> group(Set<TaskModel> tasks);
+
+  default Set<ContainerModel> group(Set<TaskModel> tasks, List<Integer> containersIds) {
+    return null;
```

Comment:
I see your point. I changed it to call the other method (basically ignoring the containers list).